emperor/support_files/js/controller.js

```diff
@@ -262,12 +263,40 @@ define([
       selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
       trigger: 'none',
       items: {
+        'saveState': {
+          name: 'Save Emperor setup',
+          icon: 'edit',
+          callback: function(key, opts) {
```

Comment:
The scope changes if this is done, and ends up as the div instead of the function. The wrapper function counteracts that.